lib/perl/Genome/Model/ClinSeq/Command/IdentifyLoh.pm

```diff
@@ -0,0 +1,280 @@
+    return;
+}
+
+#Get Varscan SNVs from somatic-variation
+sub get_varscan_snvs {
+    my $self = shift;
+    my $somvar_build = shift;
+    my $snv_prefix = shift;
+    my $dd = $somvar_build->data_directory;
+    my @varscan_snvs =
+    glob(File::Spec->join($dd, "variants/snv/varscan-somatic-*/snvs.hq.unfiltered"));
+    unless(scalar @varscan_snvs) {
+        die $self->error_message("Unable to find varscan SNVs in $dd");
+    }
+    $self->status_message("varscan snvs file found: " .
+        join("\t", @varscan_snvs));
+    $self->split_snvs($varscan_snvs[0], $snv_prefix);
+}
+
+#Get tumor/normal BAMs from somatic-variation
+sub get_tumor_normal_bam {
+    my $self = shift;
+    my $somvar_build = shift;
+    my $tumor_bam = $somvar_build->tumor_model->last_succeeded_build->
+    whole_rmdup_bam_file;
+    my $normal_bam = $somvar_build->normal_model->last_succeeded_build->
+    whole_rmdup_bam_file;
+    unless (-e $tumor_bam and -e $normal_bam) {
+        die $self->error_message("tumor_bam $tumor_bam or normal_bam " .
+            "$normal_bam does not exist.");
+    }
+    return ($tumor_bam, $normal_bam);
+}
+
+#Get the reference-sequence build from the
+# ref-align builds used in somatic-variation
+sub reference_build {
+    my $self = shift;
+    my $somvar_build = shift;
+    my $normal_refalign_build = $somvar_build->normal_build if ($somvar_build);
+    my $tumor_refalign_build = $somvar_build->tumor_build if ($somvar_build);
+    my @input_builds = ($normal_refalign_build,
+        $tumor_refalign_build, $somvar_build);
+    for my $build (@input_builds){
+        next unless $build;
+        my $m = $build->model;
+        if ($m->can("reference_sequence_build")){
+            return $m->reference_sequence_build;
+        }
+    }
+}
+
+#Filter out putative false-positive SNVs
+sub filter_snvs {
+    my $self = shift;
+    my $somvar_build = shift;
+    my $snv_prefix = shift;
+    my ($tumor_bam, $normal_bam) =
+    $self->get_tumor_normal_bam($somvar_build);
+    my $refbuild = $self->reference_build($somvar_build);
+    my $ref_fa = $refbuild->full_consensus_path('fa');
+    my $filter = Genome::Model::Tools::Varscan::SomaticFilterWorkflow->
+    create(
+        outdir => $self->outdir,
+        tumor_bam => $tumor_bam,
+        normal_bam => $normal_bam,
+        prefix => $snv_prefix,
+        reference => $ref_fa,
+    );
+    $filter->execute();
+}
+
+#Combine germline, LOH calls and sort them
+sub combine_sort_snvs {
+    my $self = shift;
+    my $snv_prefix = shift;
+    my $snv_combined = $snv_prefix . ".combined";
+    my $snv_combined_sorted = $snv_prefix . ".combined.sorted";
+    my $germline = $snv_prefix . "*.formatted.LOH.hc.filtered";
+    my $loh = $snv_prefix . "*.formatted.Germline.hc.filtered";
+    unless(scalar glob("$germline $loh")) {
+        die $self->error_message("Unable to find $germline or $loh");
+    }
+    my $cat_cmd = "cat $germline $loh > $snv_combined";
```

Comment:
I'm having a bit of trouble trying to use wild-cards(*) in either of those two, is that expected?
In the first method, the quotes result in the file not being found. In the second method, the -e operator is used, this also results in an 'input files not found' error. Is there a suggested alternative? I'm not able to clearly predict the names of my inputs here because of the variable per-chromosome files that might be generated.